quic/QuicConstants.h

```diff
@@ -159,6 +159,7 @@ enum class LocalErrorCode : uint32_t {
   INVALID_OPERATION = 0x40000017,
   STREAM_LIMIT_EXCEEDED = 0x40000018,
   CONNECTION_ABANDONED = 0x40000019,
+  CALLBACK_ALREADY_INSTALLED = 0x4000001A,
```

Comment:
Could you please add the case to the `toString` function in QuicException.cpp?

Reply:
done